luxtronik/datatypes.py

```diff
@@ -148,9 +150,9 @@ class Timestamp(Base):
 
     @classmethod
     def from_heatpump(cls, value):
-        if value > 0:
-            return datetime.datetime.fromtimestamp(value)
-        return datetime.datetime.fromtimestamp(0)
+        if value is None or value <= 0:
+            return datetime.datetime.fromtimestamp(0)
```

Comment:
Why aren't we returning None in this case?

Reply:
I would argue we should return `None` in case of invalid/unexpected values. That is more consistent with the other classes, isn't it?

Reply:
Makes sense. Adjusted.